### PR TITLE
Fix singularity docs refs

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
## Description of the Pull Request (PR):
Updates references to user-docs links that before pointed to Sylabs links.

## This fixes or addresses the following GitHub issues:
No issues refs to here.

**Before submitting a PR, make sure you have done the following:**

* Read the [Guidelines for Contributing](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
* Added changes to the CHANGELOG if necessary according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
* Added tests to validate this PR and tested this PR locally with a `make testall`
* Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
* Added myself as a contributor to the [Contributors File](https://github.com/hpcng/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: contributing.rst [hunk 1]
```diff
@@ -41,10 +41,10 @@ Raise an Issue
 
 For general bugs/issues, you can open an issue `at the GitHub repo 
 <https://github.com/hpcng/singularity/issues/new>`_. However, if you find a 
-security  related issue/problem, please email Sylabs directly at 
-`security@sylabs.io <mailto:security@sylabs.io>`_. More information about the 
-Sylabs security policies and procedures can be found `here 
-<https://www.sylabs.io/singularity/security-policy/>`__
+security  related issue/problem, please email Singularity Project Team directly at
```
Comment (DrDaveD) [inline contributing.rst]:
```suggestion
security  related issue/problem, please email the Singularity Security Team directly at 
```

File: security.rst [hunk 2]
```diff
@@ -27,13 +27,13 @@ addition to ensuring that containers are run without elevated
 privileges where appropriate, and that containers are produced by
 trusted sources, users must monitor their containers for newly
 discovered vulnerabilities and update when necessary just as they
-would with any other software. Sylabs is constantly probing to find
+would with any other software. Singularity Community is constantly probing to find
```
Comment (DrDaveD) [inline security.rst]:
```suggestion
would with any other software. The Singularity community is constantly probing to find
```

File: security.rst [hunk 3]
```diff
@@ -51,8 +51,8 @@ beyond that offered by the open source project. Security and bug-fix
 patches are backported to select versions of Singularity PRO, so that
 they can be deployed long-term where required. PRO users receive
 security fixes (without specific notification or detail) prior to
-public disclosure, as detailed in the `Sylabs Security Policy
-<https://sylabs.io/security-policy>`__.
+public disclosure, as detailed in the `Singularity Community Security Policy
```
Comment (DrDaveD) [inline security.rst:54]:
```suggestion
public disclosure, as detailed in the `Singularity Security Policy
```

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Looks good after you accept my suggested minor wording changes.

**Comment by pedroalvesbatista** [comment]
Closed accidentally

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**